src/server/saved_objects/service/lib/repository.js

```diff
@@ -286,6 +306,7 @@ export class SavedObjectsRepository {
           ...updatedAt && { updated_at: updatedAt },
           version: hit._version,
           attributes: hit._source[type],
+          migrationVersion: hit._source.migrationVersion,
```

Comment:
question: Similar to my previous question, why do we need to expose migrationVersion on things like `find`, `get`, etc.? I think only `bulkGet` is used by the export dashboard API.

Reply:
Same answer. It's probably defensive programming. I can probably remove this one. I'd have to test the import / export again, though, hopefully the CI tests would fail if those features broke!